screens/SeriesLandingScreen.tsx

```diff
@@ -106,14 +106,13 @@ interface Params {
 }
 
 function SeriesLandingScreen({ navigation, route }: Params): JSX.Element {
-
     const seriesParam = route.params?.item;
     const seriesId = route.params?.seriesId;
     const safeArea = useSafeAreaInsets();
 
     const [series, setSeries] = useState(seriesParam);
     const [contentFills, setContentFills] = useState(false);
-    const [videos, setVideos] = useState<VideoData>();
+    const [videos, setVideos] = useState<VideoData | any>();
```

Comment:
I understand that this project moves quickly, but we shouldn't be increasing technical debt with weak typing. @itbel please, where possible, don't use `any`.

Reply:
Is there a shared type.ts? I was just looking at creating one to replace any->CognitoUser (which I just introduced!) but that I need to extend...

Reply:
found it...never mind...

Reply:
There isn't a `types.ts` file (yet) but definitely a good idea

Reply:
I struggled to apply the correct typing to this one in particular and felt that I was spending too much time on it. I'm not sure how to type this. Any pointers would be appreciated.

Reply:
@itbel `Array<NonNullable<NonNullable<NonNullable<NonNullable<GetCustomPlaylistQuery['getCustomPlaylist']>['videos']>['items']>[0]>['video']>;` is how I would type data from custom playlists.

Then just add an or operator to the original type.

```
type VideoData =
  | NonNullable<NonNullable<GetSeriesQuery['getSeries']>['videos']>['items']
  | Array<NonNullable<NonNullable<NonNullable<NonNullable<GetCustomPlaylistQuery['getCustomPlaylist']>['videos']>['items']>[0]>['video']>
```

For this to work you need to add a nullish coalescing operator to the `setVideos()` call:
```javascript
setVideos(
  json.data?.getCustomPlaylist?.videos?.items?.map((item) => {
    return item?.video ?? null;
  })
);
```

Reply:
Thank you so much for this. I will be submitting another PR with this change.